plugins/org.eclipse.reddeer.swt/src/org/eclipse/reddeer/swt/impl/table/AbstractTable.java

```diff
@@ -196,7 +196,7 @@ public void deselectAll() {
 	}
 
 	private void waitUntilTableHasRows() {
-		new WaitUntil(new TableHasRows(this), TimePeriod.DEFAULT, false);
+		new WaitUntil(new TableHasRows(this), TimePeriod.SHORT, false);
```

Comment:
IMHO yes, if somebody has a table which takes longer to init data, then he should use WaitUntil in his code. 
If I have a table which has 0 items and I will call table.getItems() I will always have to wait for DEFAULT time period (3s) which is quite long.